src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -154,9 +154,12 @@ class WorkspaceMembersPage extends React.Component {
 
     /**
      * Add or remove all users from the selectedEmployees list
+     * @param {Object} memberList
      */
-    toggleAllUsers() {
-        let policyMemberList = lodashGet(this.props, 'policyMemberList', {});
+    toggleAllUsers(memberList) {
+        // If a memberList is provided, only select those members. Otherwise, select all the policy ones
+        let policyMemberList = _.isEmpty(memberList) ? lodashGet(this.props, 'policyMemberList', {}) : memberList;
+
```

Comment:
I'm not sure would why would we need this. There is a valid case where `memberList` can be empty that's when the workspace has an owner alone. Getting value from the prop is unnecessary,